<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regulateur_temperature</class>
 <widget class="QWidget" name="Regulateur_temperature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridWidget_11" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 174, 156);
border-radius: 40px;</string>
   </property>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Temperature Actuelle&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(0, 143, 105);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(0, 143, 105);
}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridWidget_10" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 174, 156);
border-radius: 40px;</string>
   </property>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Saisir temperature ideale&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(0, 143, 105);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(0, 143, 105);
}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(121, 99, 90);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(121, 99, 90);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moins">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(121, 99, 90);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(121, 99, 90);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridWidget_12" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 174, 156);
border-radius: 40px;</string>
   </property>
   <widget class="QPushButton" name="supprimer_15">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(0, 143, 105);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(0, 143, 105);
}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commencer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(0, 143, 105);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(0, 143, 105);
}</string>
    </property>
    <property name="text">
     <string>Mise en marche </string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView_date">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimer_date">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(237, 218, 204);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
border-radius: 14px;

	border-style: outset;
    border-width: 2px;
    border-radius: 14px;
    border-color: beige;
  
 
    padding: 0px;
	
	
	
	font: 10pt &quot;Franklin Gothic Heavy&quot;;
}
QPushButton:hover
{
background-color: rgb(0, 143, 105);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color: rgb(0, 143, 105);
}</string>
   </property>
   <property name="text">
    <string>Tout supprimer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
